tests/testthat/test_PlainTextParam.R

```diff
@@ -82,16 +137,39 @@ test_that("storeResults,loadResults,PlainTextParam,XcmsExperiment works", {
     expect_true(file.exists(file.path(param@path, "rtime_adjusted.txt")))
     expect_true(file.exists(file.path(param@path, "feature_definitions.txt")))
     expect_true(file.exists(file.path(param@path, "feature_peak_index.txt")))
-    ##load data again
+
+    ## load data again
     load_xmse <- loadResults(object = XcmsExperiment(), param)
     expect_true(inherits(load_xmse, "XcmsExperiment"))
-    expect_equal(xmse_full@featureDefinitions,
+    expect_equal(tmp@featureDefinitions,
                  load_xmse@featureDefinitions)
-    expect_equal(featureValues(xmse_full), featureValues(load_xmse))
-    expect_equal(adjustedRtime(xmse_full), adjustedRtime(load_xmse))
+    expect_equal(featureValues(tmp), featureValues(load_xmse))
+    expect_equal(adjustedRtime(tmp), adjustedRtime(load_xmse))
     expect_no_error(filterRt(load_xmse, c(3000, 3500)))
-    ## not sure how to check for the processHistory slot
-    ## still not sure how to create UT for spectraPath
-})
+    expect_equal(tmp@chromPeaks, load_xmse@chromPeaks)
+    expect_equal(tmp@chromPeakData, load_xmse@chromPeakData)
+    expect_equal(tmp@sampleData, load_xmse@sampleData)
+    expect_equal(length(tmp@processHistory), length(load_xmse@processHistory))
+    expect_equal(tmp@processHistory[[1L]], load_xmse@processHistory[[1L]])
+    expect_equal(tmp@processHistory[[2L]], load_xmse@processHistory[[2L]])
+    expect_equal(tmp@processHistory[[3L]], load_xmse@processHistory[[3L]])
+    expect_equal(tmp@processHistory[[4L]], load_xmse@processHistory[[4L]])
+    expect_equal(tmp@processHistory[[5L]], load_xmse@processHistory[[5L]])
+    ## The 6th param object contains functions for which the comparison fails
```

Comment:
damn   it, i tried to do that but obviously only tested it on the 6th param and gave up when I saw the error :')